src/services/workspace-client/devWorkspaceClient.ts

```diff
@@ -67,7 +67,7 @@ export class DevWorkspaceClient extends WorkspaceClient {
     const workspaces = await this.dwApi.listInNamespace(defaultNamespace);
     const availableWorkspaces: IDevWorkspace[] = [];
     for (const workspace of workspaces) {
-      if (!isDeleting(workspace) && !isWebTerminal(workspace)) {
```

Comment:
The import needs to be cleaned up, after you remove usage. Actually, isDeleting func is totally unused now.

Reply:
I have fixed it

Reply:
https://github.com/eclipse-che/che-dashboard/pull/225/files#diff-bd3e628ea1cf6067bd9ad9b7dd8509178ad6b95d2593f3f5affd3e673f01033cL19